src/features/auth/ui/SignInSocialLogin.tsx

```
@@ -0,0 +1,45 @@
+'use client';
+import useSocialLogin from '../hooks/useSocialLogin';
+
+const SignInSocialLogin = () => {
+  const { handleSocialLogin } = useSocialLogin();
+  return (
+    <div className="space-y-3">
+      <button
+        className="w-full flex items-center justify-center py-3 px-4 bg-gray-800/50 hover:bg-white/5 border border-gray-700 rounded-[10px] text-white transition-all duration-200"
+        onClick={() => handleSocialLogin('google')}
+      >
+        <svg className="w-5 h-5 mr-3" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)